preferences.yaml

```diff
@@ -0,0 +1,67 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Preferences API
+  description: |-
+    The API lets you manage attributes associated with users (user's preferences).
+    You can get any attribute for any user (read only) according to the rights you owe on the user resource.
+    There rights are set up at moment you've subscribed the service exposed by this API.
+  license:
+    name: MIT
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/users/{fiscal_code}/preferences':
```

Comment:
`preferences` is not a nested resource, compare this to `/users/{fiscal_code}/messages`